docs/topics/installing-cli-tool.adoc

```diff
@@ -41,14 +42,27 @@ To install using the downloadable `.zip` file:
 +
 When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with the actual path to your {ProductShortName} installation.
 
-////
+
 [id="installing-using-podman_{context}"]
 == Installing the {CLINameTitle} using Podman
 
+.Prerequisites
+
+* Red Hat Container Registry Authentication for `registry.redhat.io`. Red Hat distributes container images from  registry.redhat.io, which requires authentication. For more details, see link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
```

Comment:
There are two whitespaces between "from" and "registry.redhat.io", is that expected?

The same thing happens in line 15

Reply:
@abrugaro - WOW! You have eyes like a hawk!
Apologies, i was going too quickly and missed it